layouts/articles/single.html

```diff
@@ -66,50 +70,99 @@ <h5 class="text-16 text-blockquote">{{ .Date.Format "January 2, 2006" }}</h5>
           <a class="social-list-facebook" href=""></a>
         </div>
         <div class="social-list">
-            <div class="addthis_inline_share_toolbox"></div>
-        </div>
-        <div class="section-rich-text-leading">
-        {{ .Content }}
+          <div class="addthis_inline_share_toolbox"></div>
         </div>
+        <div class="section-rich-text-leading">{{ .Content }}</div>
 
         {{ if hugo.Environment | eq "development" }}
 
-        <script type = "text/javascript">
-          var images = document.getElementsByTagName('img'); 
+        <script type="text/javascript">
+          var images = document.getElementsByTagName("img");
           for (var i = 0; i < images.length; i++) {
-            images[i].src = images[i].src.replace("/engineering-education/", "/");
-          };
+            images[i].src = images[i].src.replace(
+              "/engineering-education/",
+              "/"
+            );
+          }
           console.log(images[2].src);
         </script>
 
-        {{ else }}
-        {{ end }}
+        {{ else }} {{ end }}
       </div>
     </div>
   </section>
   <div class="comments-wrapper">
     <div id="hyvor-talk-view"></div>
   </div>
-</article>
 
+  {{ $related := .Site.RegularPages.RelatedIndices . "topics" | first 3 }} 
+  {{ with $related }}
+
+  <section class="resource-block section-container xs-mb-40 xs-mt-40 relative">
+    <div class="resource-block-left-large absolute"></div>
+    <div class="resource-block-left-stripes absolute"></div>
+    <div class="resource-block-header text-center">
+      <h2 class="title-3 xs-mb-40 text-link">Similar Articles</h2>
+    </div>
+    <div class="resource-block-inner flex flex-wrap justify-center">
+
+      {{ range . }}
+      <a class="decoration-none resource-block-item" href="{{ .RelPermalink }}">
+        <div class="resource-block-item-wrap fill-white flex cta-shadow flex-column">
+          {{ if isset .Params "images" }}
+            {{ if (gt (len .Params.images) 0) }}
+              {{ if (index .Params.images 0).url }}
+                <img src="{{ (index .Params.images 0).url }}" {{ if (index .Params.images 0).alt}} alt="{{ (index .Params.images 0).alt }}"{{ end }}/>
```

Comment:
@louisefindlay23 the issue here with the url being different on local vs prod can be fixed by adding this bit of logic: 
`<img src="{{ if eq $.Site.Params.env "local" }}{{ replace (index .Params.images 0).url "/engineering-education" "" }}{{ else }}{{ (index .Params.images 0).url }}{{ end }}" {{ if (index .Params.images 0).alt}} alt="{{ (index .Params.images 0).alt }}"{{ end }}/>`